legacy/pom.xml

```diff
@@ -75,6 +75,7 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk-iam</artifactId>
+      <version>${aws-java-sdk.version}</version>
```

Comment:
I think this is in error.

https://jira.pentaho.com/browse/PDI-17867 Updated the version globally in the parent pom.

See:  https://github.com/pentaho/maven-parent-poms/pull/112/files